scripts/default/opensearch/build.sh

```diff
@@ -77,3 +77,15 @@ distributions="$(dirname "${zipPath}")"
 echo "COPY ${distributions}/*.zip"
 mkdir -p $OUTPUT/plugins
 cp ${distributions}/*.zip ./$OUTPUT/plugins
+
+#This is to check if a component has a plugin opensearch.zippublish applied and run publishMavenzipPublicationToZipstagingRepository.
+MAVEN_ZIP_PUBLISH_TASK="publishMavenzipPublicationToZipstagingRepository"
```

Comment:
Can this be renamed to `publishZipsToMavenRepository` or something similar? Much simpler and precise.

Reply:
@gaiksaya 
so the reason I have added this way is this var MAVEN_ZIP_PUBLISH_TASK is actually referenced same in the java code, https://github.com/opensearch-project/OpenSearch/pull/2988/files#diff-d56f11b358bf18f6e6c28d2a528aa08d4436d01ce9b3c3c45bab1fa970d9cb43R49 and this is the format `publishMavenzipPublicationToZipstagingRepository` gradle expects for publishing a custom repo (mavenzip) to custom staging repo (zipstaging).

Reply:
just to add this task is coming from the custom gradle plugin (identifies once the plugin is applied).

Reply:
I see, Yeah I thought this can be changed in upstream PR as well but looks like we can't since its a gradle requirement.